pom.xml

```diff
@@ -36,14 +36,6 @@
     </license>
   </licenses>
   
-  <dependencies>
-    <dependency>
-      <groupId>commons-lang</groupId>
-      <artifactId>commons-lang</artifactId>
-      <version>[2.3,]</version>
-    </dependency>
-  </dependencies>
-  

```

Comment:
I removed this as this prevents me running `mvn eclipse:eclipse`.
Anyway, jenkins-core depends on commons-lang and I don't need to write dependency in each plugins as they can access commons-lang via the dependency of jenkins-core.